types/firestore.d.ts

```diff
@@ -416,7 +416,7 @@ declare namespace FirebaseFirestore {
    *
    * @class
    */
-  export class BulkWriter {
+  class BulkWriter {
```

Comment:
All types in a `d.ts` file as exported by default. Unfortunately, we can only hide them by removing the entries completely.

Reply:
done.